### ISSUE TITLE
detach operation of xyz

### ISSUE THREAD

**Issue by tb2-sy** [markomih/SplatFields#10]
Thanks for your nice work! 
https://github.com/markomih/SplatFields/blob/main/train.py#L53 
Why is xyz detached in the code? If I understand correctly, this would mean that the position of the point cannot be optimized. If the Gaussian points are randomly initialized, wouldn’t this make the optimization process more challenging?

**Comment by markomih** [comment]
Hi @tb2-sy,

Thank you for your kind words!

The xyz positions are detached in the code to ensure stability during the optimization process. Without detachment, the locations would be directly optimized via gradient descent, bypassing the gradients from the neural field architecture, which could lead to instability.

That said, the xyz positions are still indirectly optimized through adaptive density control and can also be further refined by the flow MLP.

Best regards,
Marko
